B1_HEIGHT: 0.642
B1_WIDTH: 0.7
CELL_HEIGHT:
  DDG: 0.505
  SDG: 0.48
  SSG: 0.48
CELL_WIDTH: 
  DDG: 0.705
  SDG: 0.705
  SSG: 0.68
CONFIDENCE_RATIO:
- 0.95
- 0.95
- 0.9
- 0.85
- 0.8
- 0.75
- 0.65
- 0.55
- 0.45
- 0.35
- 0.25
EPSILON: 0.035
FRAME_PROCESSING: 20
GRID_TYPE:
- SSG
- SDG
- DDG
HEIGHT_HIGH_THRE: 0.9
HEIGHT_LOW_THRE: 0.6
MIN_CLUSTER_SIZE: 15
NEIGHBOR_DISTANCE: 0.044
NEIGHBOR_LINE: 0.03
R5_BIN_HEIGHT: 0.96
R5_HEIGHT: 0.505
R5_WIDTH: 0.7
WIDTH_B1_THRE: 0.653
WIDTH_R5_THRE: 0.653
X_OFFSET: 0.3285309549275511
X_RATIO: 0.9685697418715534
Y_OFFSET_L: 0.00046626801067171204
Y_OFFSET_R: 0.04514637895445253
Y_RATION_L: 0.983836292499241
Y_RATION_R: 0.9966073445665985
